srcpkgs/zsnes/template

```diff
@@ -2,22 +2,22 @@
 #
 # 32bit
 archs="i686"
-lib32mode="full"
 wrksrc="zsnes_1_51"
 build_wrksrc="src"
+lib32mode="full"
 
 pkgname=zsnes
 version=1.51
-revision=3
+revision=4
 build_style=gnu-configure
```

Comment:
Please group these together, it will bubble up a bunch of xlints.

Reply:
The only template that has a mis-located lib32mode is zsnes. Of the other six templates containing a lib32mode= line (doom3, mesa, pcsx2, qt5, swars, and wine), only doom3, qt5, and swars have lint errors and none of them complain about location.

Or are you saying to consolidate the lines of the zsnes template across the board? Entirely doable but I'd like to know that's the ask before redoing it.

Reply:
Yes, only fix the lines in the zsnes template itself. That break between the two blocks makes xlint not do a full lint.

Reply:
Cool thanks, fixed.